src/deluge/gui/views/view.cpp

```diff
@@ -1122,6 +1122,8 @@ void View::setKnobIndicatorLevel(uint8_t whichModEncoder) {
 
 	// Quantized Stutter FX
 	if (modelStackWithParam->paramId == Param::Unpatched::STUTTER_RATE
```

Comment:
Redundant - have the next checks do modelStackWithParam->isParam(unpatched sound/global, Param::Unpatched::STUTTER_RATE) instead